libraries/metadata_util.rb

```diff
@@ -7,14 +7,29 @@ class MetadataUtil
     def initialize(volume)
       mdutil_possible_states = { 'Indexing enabled.' => ['on', ''],
                                  'Indexing disabled.' => ['off', ''],
-                                 'Indexing and searching disabled.' => ['off', '-d'] }
+                                 'Indexing and searching disabled.' => ['off', '-d'],
+                                 'Spotlight server is disabled.' => %w(dis dis) }
```

Comment:
I'm not sure that this belongs in the states table, because it's kind of an exceptional case. I think it would be clearer if `server_disabled?` just parsed `mdutil_output` to determine if this case is true.

Reply:
Changes have been added to the next set of commits.